rasgotransforms/rasgotransforms/transforms/histogram/histogram.sql

```diff
@@ -11,9 +11,19 @@ SELECT
   ,COUNT(1) AS REC_CT
 FROM
   (SELECT {{ column }}::float AS COL FROM {{ source_table }}
-  {%- if filter_statements is iterable -%}
-  {%- for filter_statement in filter_statements %}
-  {{ 'WHERE' if loop.first else 'AND' }} {{ filter_statement }}
+  {%- if filters is defined and filters %}
+  {% for filter_block in filters %}
+  {%- set oloop = loop -%}
+  {{ 'WHERE ' if oloop.first else ' AND ' }}
+  {%- if filter_block is not mapping -%}
+  {{ filter_block }}
+  {%- else -%}
+      {%- if filter_block['operator'] == 'CONTAINS' -%}
+  {{ filter_block['operator'] }}({{ filter_block['columnName'] }}, {{ filter_block['comparisonValue'] }})
+      {%- else -%}
+  {{ filter_block['columnName'] }} {{ filter_block['operator'] }} {{ filter_block['comparisonValue'] }}
+      {%- endif -%}
+  {%- endif -%}
```

Comment:
```suggestion
  {%- if filters is defined and filters %}
      {% for filter_block in filters %}
          {%- set oloop = loop -%}
          {{ 'WHERE ' if oloop.first else ' AND ' }}
          {%- if filter_block is not mapping -%}
              {{ filter_block }}
          {%- else -%}
              {%- if filter_block['operator'] == 'CONTAINS' -%}
                  {{ filter_block['operator'] }}({{ filter_block['columnName'] }}, {{ filter_block['comparisonValue'] }})
               {%- else -%}
                  {{ filter_block['columnName'] }} {{ filter_block['operator'] }} {{ filter_block['comparisonValue'] }}
               {%- endif -%}
  {%- endif -%}
```

Would be nice to update these so the indentation matches the logic around it

Reply:
@ChrisGriffithRASGO check it now?